proxy/jsonrpc/proxy_test.go

```diff
@@ -0,0 +1,39 @@
+package jsonrpc_test
```

Comment:
why do you need a different package here? `jsonrpc` is not good enough?

Reply:
It's better to have a test package since we don't need to access any internals.